zerver/tests/test_signup.py

```diff
@@ -1302,6 +1311,28 @@ def test_successful_delete_invitation(self) -> None:
                           lambda: ScheduledEmail.objects.get(address__iexact=invitee,
                                                              type=ScheduledEmail.INVITATION_REMINDER))
 
+    def test_delete_multiuse_invite(self) -> None:
+        """
+        A DELETE call to /json/invites/multiuse<ID> should delete the
+        multiuse_invite.
+        """
+        self.login(self.example_email("iago"))
+
+        zulip_realm = get_realm("zulip")
+        multiuse_invite = MultiuseInvite.objects.create(referred_by=self.example_user("hamlet"), realm=zulip_realm)
+        create_confirmation_link(multiuse_invite, zulip_realm.host, Confirmation.MULTIUSE_INVITE)
+        result = self.client_delete('/json/invites/multiuse/' + str(multiuse_invite.id))
+        self.assertEqual(result.status_code, 200)
+        error_result = self.client_delete('/json/invites/multiuse/' + str(multiuse_invite.id))
+        self.assert_json_error(error_result, "No such invitation")
+
+        # Test deleting multiuse invite from another realm
+        mit_realm = get_realm("zephyr")
+        multiuse_invite_in_mit = MultiuseInvite.objects.create(referred_by=self.mit_user("sipbtest"), realm=mit_realm)
+        create_confirmation_link(multiuse_invite_in_mit, mit_realm.host, Confirmation.MULTIUSE_INVITE)
+        error_result = self.client_delete('/json/invites/multiuse/' + str(multiuse_invite_in_mit.id))
+        self.assert_json_error(error_result, "No such invitation")
```

Comment:
for any test of this form, you really want to include a test block for successfully deleting the thing, that also verifies it's no longer in the database if you query for it.  While you've gotten 100% coverage here, we don't actually check that the this API works at all here.

Reply:
(I guess you sorta have that above with the double-delete block, but we should have an explicit assert that the MultiuseInvite object doesn't still exist)